tests/test_scheduler.py

```
@@ -1643,49 +1062,708 @@ def test_inverted_mode_with_exclusion(self, now, today_prices):
+        """With enough data, multiple on-blocks are scheduled iteratively."""
+        last_on = now - timedelta(hours=2)
+        schedule = build_minimum_runtime_schedule(
+            raw_today=today_prices,
+            raw_tomorrow=tomorrow_prices,
+            min_hours_on=1.0,  # 1h on-blocks
+            now=now,
+            max_hours_off=4.0,  # Must run every 4 hours
+            last_on_time=last_on,
+        )
+
+        active = [s for s in schedule if s["status"] == "active"]
```

Ruff (0.15.2)

[error] Local variable `active` is assigned to but never used

Remove assignment to unused variable `active`

(F841)